pkg/config/setup/config.go

```diff
@@ -1086,6 +1086,11 @@ func InitConfig(config pkgconfigmodel.Config) {
 	// The histogram buckets use to track the time in nanoseconds it takes for a DogStatsD listeners to push data to the server
 	config.BindEnvAndSetDefault("telemetry.dogstatsd.listeners_channel_latency_buckets", []string{})
 
+	// Agent Telemetry
```

Comment:
Please mention here that the feature is experimental and should not be enabled unless prompted by Datadog Support

Reply:
Agree, fixed in the last commit